src/openassetio-core/include/openassetio/hostApi/Manager.hpp

```diff
@@ -582,15 +582,15 @@ class OPENASSETIO_CORE_EXPORT Manager {
    * corresponding index of the entity reference in @p entityReferences
    * along with a boolean indicating existence, as defined above. The
    * callback will be called on the same thread that initiated the call
-   * to `resolve`.
+   * to `entityExists`.
```

Comment:
[consider][minor] Good spot, ideally would be a separate commit, but probably not worth the effort.

Reply:
Not worth the effort :+1: